components/PostCardList/PostCardList.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import styled from "styled-components";
+import { MOBILE_MEDIA } from "../../constants/mediaquery";
+import PostTypeContext, { PostType } from "./PostTypeContext";
+
+interface Props extends React.Attributes {
+  postType?: PostType;
```

Comment:
I'm not 100% sure but I think I mentioned about this prop in the previous review. `PostType` sounds "kind of posts". Could you come up with some proper name for it?